lib/ansible/cli/doc.py

```diff
@@ -312,10 +312,10 @@ class DocCLI(CLI, RoleMixin):
     _RULER = re.compile(r"\bHORIZONTALLINE\b")
 
     # rst specific
-    _REFTAG = re.compile(r":ref:")
-    _TERM = re.compile(r":term:")
-    _NOTES = re.compile(r".. note:")
-    _SEEALSO = re.compile(r"^\s*.. seealso:.*$", re.MULTILINE)
+    _RST_NOTE = re.compile(r".. note::")
+    _RST_SEEALSO = re.compile(r".. seealso::")
+    _RST_REFTAGS = re.compile(r":\w+?:`")
+    _RST_GENTAGS = re.compile(r".. \w+?::")
```

Comment:
Let's keep the naming aligned with docutils' own definitions, at least:
```suggestion
    _RST_REF_ROLE = re.compile(r":\w+?:`")
    _RST_DIRECTIVE = re.compile(r".. \w+?::")
```